src/domain/bitcoin/lightning/index.types.d.ts

```diff
@@ -8,6 +8,9 @@ type InvoiceExpiration = Date & { [invoiceExpirationSymbol]: never }
 declare const encodedPaymentRequestSymbol: unique symbol
 type EncodedPaymentRequest = string & { [encodedPaymentRequestSymbol]: never }
 
+declare const paymentIdSymbol: unique symbol
+type PaymentId = string & { [paymentIdSymbol]: never }
```

Comment:
I feel `PaymentId` might be too vague given the thing it can mean multiple things. Would `PaymentLedgerId` be a better name?

Reply:
Yea that makes sense